dali/operators/reader/loader/webdataset/tar_utils_test.cc

```diff
@@ -70,25 +71,63 @@ TEST(LibTarUtilsTestSimple, LongNameIndexing) {
   std::string name_prefix(128, '#');
   for (int idx = 0; idx < 1000; idx++) {
     ASSERT_EQ(archive.GetFileName(), name_prefix + to_string(idx));
+    ASSERT_EQ(archive.GetFileType(), TarArchive::ENTRY_FILE);
     archive.NextFile();
   }
 }
 
-void TestArchiveEntries(TarArchive& archive, const std::vector<std::string>& prefixes,
-                        int beg, int end, bool preread) {
+TEST(LibTarUtilsTestSimple, Types) {
+  std::string filepath(dali::filesystem::join_path(std::getenv("DALI_EXTRA_PATH"),
+                                                   "db/webdataset/sample-tar/types.tar"));
+  TarArchive::EntryType types[14] = {
```

Comment:
That's a nitpick:
```c++
std::vector<TarArchive::EntryType> types = {...
```
And then you can use it in `for` below:
```c++
for (int i=0; i < types.size(); i++)
```
So when it so happens, that something has to be added or removed from `types`, there be a single place of change, no 3 of them...

But whatever you prefer ;)

Reply:
done